package-lock.json

```diff
@@ -1,8 +1,741 @@
 {
   "name": "lorawan-devices",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Revert this file and respect tooling dependencies (and their version) in `README`